.github/workflows/tempate-sync-pat.yml

```
@@ -0,0 +1,26 @@
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+
+      - name: actions-template-sync
+        uses: AndreasAugustin/actions-template-sync@v2
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          source_repo_path: VilnaCRM-Org/php-service-template
+          upstream_branch: main
+          pr_labels: template_sync
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)